inlang/source-code/website/src/interface/components/Card.tsx

```diff
@@ -128,7 +128,11 @@ export default function Card(props: { item: any; displayName: string }) {
 									</Show>
 								</div>
 								<div class="flex flex-col justify-between items-start">
-									<p class="m-0 mb-2 text-sm text-surface-800 line-clamp-1 leading-none no-underline font-semibold group-hover:text-surface-900 transition-colors">
+									<p
+										// eslint-disable-next-line solid/style-prop
+										style="text-wrap: balance;"
```

Comment:
Isn't `text-balance` already a valid tailwind class?
See here: https://github.com/tailwindlabs/tailwindcss/pull/11320

If not, nvm, good improvement!

Reply:
It didn't suggest it to me in the IDE, so i just assumed it wasn't. I'll double-check

Reply:
`text-balance` is not yet a utility, it's coming in the upcoming version of tailwind

Reply:
@LorisSigrist ahh okay my bad, approving the changes then!